rest/rest-client/src/test/java/functionaltests/SchedulerClientTest.java

```diff
@@ -165,6 +165,14 @@ public void testJobSubmissionEventListener() throws Exception {
         }
         client.removeEventListener();
     }
+    
+    @Test
+    public void testGetJobContent() throws Exception {
+    	ISchedulerClient client = clientInstance();
+    	Job job = defaultJob();
+        JobId jobId = client.submit(job);
```

Comment:
weird, i dont have this extra space in my code under eclipse

`@Test
    public void testGetJobContent() throws Exception {
    	ISchedulerClient client = clientInstance();
    	Job job = defaultJob();
        JobId jobId = client.submit(job);
        TaskFlowJob content = (TaskFlowJob) client.getInitialJobContent(jobId);
    }`